packages/web3-common/src/web3_base_provider.ts

```diff
@@ -51,6 +51,27 @@ export abstract class Web3BaseProvider<API extends Web3APISpec = EthExecutionAPI
 	public abstract getStatus(): Web3BaseProviderStatus;
 	public abstract supportsSubscriptions(): boolean;
 
+	/**
+	 * @deprecated Please use `.request` instead.
+	 *
+	 * @param payload - Request Payload
+	 * @param cb - Callback
+	 */
+	public send<Method extends Web3APIMethod<API>, ResponseType = Web3APIReturnType<API, Method>>(
```

Comment:
create `sendAsync` as well for supporting legacy apps https://eips.ethereum.org/EIPS/eip-1193#sendasync-deprecated

Reply:
Done.